trainers/gan_trainer.py

```diff
@@ -96,8 +96,8 @@ def train(self):
             g_cost = -g_loss
             self.g_optimizer.step()
 
-            # TODO: Log to WandB
-            # logger.info(f'Generator iteration: {g_iter}/{self.model.generator_iters}, g_loss: {g_loss}')
+            # Log to WandB
+            wandb.log({'g_loss': g_loss})
```

Comment:
I think we have to either log all at once or use the step argument so the x axis is the epochs/batch
see https://docs.wandb.ai/library/log#incremental-logging

Reply:
I tried to use the step argument actually! If you look at commit my commit history, the last commit removes the step argument because the logging was ignored by WandB since the steps were not strictly going up. (for example the steps of the discriminator loops.) Not sure what do to here.

Reply:
Ok now i'm directly logging the generator and discriminator iterations! I think it's the best way to go.

Reply:
Result with that here : https://wandb.ai/game-theory/MO-DAGAN/runs/u3cyc391